# Microbial_Pathogens_on_C.elegans
##### This repository contains code used for quality control and data analysis presented in:

> **Abby Kaufman, Jonathan G. Miller, Ray A. Enke, Kristopher L. Schmidt. Transcriptome dataset analyzing the innate immune responses of C. elegans to microbial pathogens.**


## Data Availability
##### Data is available in NCBI SRA under the accession number 


## Software Used

| Software | Version | URL |
| --- | --- | --- |
| FastQC | 0.11.5 | http://www.bioinformatics.babraham.ac.uk/projects/fastqc/ |
| MultiQC | 1.7 | https://multiqc.info |
| HISAT2 | 2.1.0 | http://daehwankimlab.github.io/hisat2/ |
| featureCounts | 1.6.0 | https://subread.sourceforge.net/featureCounts.html#:~:text=featureCounts%20is%20a%20highly%20efficient,and%20genomic%20DNA%2Dseq%20reads. |
| edgeR | 2.0 | https://bioconductor.org/packages/release/bioc/html/edgeR.html |

## Data analysis walkthroughs & code

Walkthroughs and code used for all of the quality assessment and data analysis steps are available in each of the below links.

1. [Quality assessment with FastQC](https://github.com/JonathanGM70/Microbial_Pathogens_on_C.elegans/blob/main/Walkthroughs_and_Code/FastQC)
2. [Quality analysis summary with MultiQC](https://github.com/JonathanGM70/Microbial_Pathogens_on_C.elegans/blob/main/Walkthroughs_and_Code/MultiQC)
3. [Alignment to reference genome using HISAT2](https://github.com/JonathanGM70/Microbial_Pathogens_on_C.elegans/blob/main/Walkthroughs_and_Code/HISAT2)
4. [Aligned exonic counts with featureCounts](https://github.com/JonathanGM70/Microbial_Pathogens_on_C.elegans/blob/main/Walkthroughs_and_Code/featureCounts)
5. [Differential expression analysis using edgeR](https://github.com/JonathanGM70/Microbial_Pathogens_on_C.elegans/blob/main/Walkthroughs_and_Code/edgeR)
